00-introduction.md

```diff
@@ -21,6 +21,7 @@ This is version 0.
 7. [BOLT #7](07-routing-gossip.md): P2P Node and Channel Discovery
 8. [BOLT #8](08-transport.md): Encrypted and Authenticated Transport
 9. [BOLT #9](09-features.md): Assigned Feature Flags
+11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
```

Comment:
> Lightning is part of Bitcoin, therefore should be using the BIP process.

Additional features that happen to take advantage of Bitcoin's blockchain are not Bitcoin, not a part of Bitcoin, and should not be BIP's.

Otherwise, Blockchain.info, coinbase, BU, classic, and other companies/interests have a legitimate right to submit BIPs for their own systems developed on top of Bitcoin.  BIP is for things that exclusively affect and relate to Bitcoin's core architecture and consensus.